graphein/protein/visualisation.py

```diff
@@ -239,6 +243,21 @@ def plotly_protein_structure_graph(
         G, colour_map=edge_color_map, colour_by=colour_edges_by
     )
 
+    # Get node size 
+    def node_scale_by(G, feature):
+        if feature == 'degree':
+            return lambda k : node_size_min + node_size_multiplier * G.degree[k]
+        elif feature == 'rsa':
+            return lambda k : node_size_min + node_size_multiplier * G.nodes(data=True)[k]['rsa']
+        # Meiler embedding dimension
+        p = re.compile("meiler-([1-7])")
+        if dim := p.search(feature).group(1):
```

Comment:
Could you remove the walrus operator here? We're trying to maintain support for python 3.7 so Graphein can be used in Colab.

Reply:
@a-r-j I've taken it out, how's that?